### ISSUE TITLE
Bug: Input - VoiceOver doesn't announce Character Counter

### ISSUE THREAD

**Issue by nikitaorliak-cengage** [cengage/react-magma#1682]
**Component**
Input - Character Counter

**Describe the bug**
VoiceOver doesn't read the character counter when it's updated 

**To Reproduce**
1. Go to the `Input` component
2. `Character Counter` section
3. Turn on VoiceOver
4. Type anything in the input
5. See that VoiceOver doesn't announce the character counter

**Expected behavior**
VoiceOver announces it on change

**Screenshots**
![Image](https://github.com/user-attachments/assets/914acdba-e911-4e42-a83a-ce88134d23c0)

**Desktop (please complete the following information):**
 - MacOs (Sonoma, Sequoia)
 - Safari, Firefox, Chrome

**Comment by orion-cengage** [comment]
Tested in Chrome, Firefox and Safari with Voiceover. Everything is working as expected.
